runtime/runtime.go

```diff
@@ -81,13 +84,20 @@ func NewMetadata(data string) *Metadata {
 }
 
 func (d *Default) Reset(cluster *v1.Cluster) error {
-	panic("implement me")
+	return d.reset(cluster)
```

Comment:
The reset master and reset node will also implement in this interface, not only registry.